modules/slack/blocks/block_kit_examples/channel_join_request_blocks.json

```diff
@@ -0,0 +1,29 @@
+{
+	"blocks": [
+		{
+			"type": "section",
+			"block_id": "request_main",
+			"text": {
+				"type": "mrkdwn",
+				"text": ":tada: <google.com|@JulioMendez> has requested to join the channel."
```

Comment:
Is this a placeholder for a real user link or does slack have some magic strings that it automatically replaces these hard-coded values with the real user name and id?

Reply:
Oh I didn't realize at first this is an example. Is this for the docs?

Reply:
Yup exactly - really mostly to help me without having to look back at logs.